src/programmatic_policy_learning/envs/providers/pr_bench.py

```diff
@@ -0,0 +1,23 @@
+"""PRBench environment provider."""
+
+import prbench
+from gymnasium import spaces
+
+from programmatic_policy_learning.envs.utils.wrappers import patch_box_float32
+
+
+def pr_bench(env_config):
```

Comment:
maybe we should call this and also `ggg` something else, like `prbench_provider`? to avoid confusion with the modules themselves